devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -44,6 +44,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0009,0020,0101,0402,0B05,FDBD", outClusters: "000A,0019", manufacturer: "Kwikset", model: "SMARTCODE_DEADBOLT_10T", deviceJoinName: "Kwikset 10-Button Touch Deadbolt"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0101", manufacturer:"Kwikset", model:"Smartcode", deviceJoinName: "Kwikset Smartcode Lock"
 		fingerprint profileId: "0104", inClusters: "0000, 0001, 0003, 0009, 0020, 0101, 0B05, FC00", outClusters: "000A, 0019", manufacturer: "Schlage", model: "BE468", deviceJoinName: "Schlage Connect Smart Deadbolt"
+		fingerprint profileId: "0104,000A", inClusters: "0000,0001,0003,0009,000A,0101,0020,0B05", outClusters: "000A,0019", manufacturer: "Yale", model: "YDD-D4F0 TSDB", deviceJoinName: "Lock wood Smart Lock"
```

Comment:
There can only be a single profile which is usually `0104`

Reply:
I'm not familiar with this device but according to the ticket "Lockwood" should be one word

Reply:
This is the raw description:
01 0104 000A 00 08 0000 0001 0003 0009 000A 0101 0020 0B05 02 000A 0019

Yale products are sold through ASSA ABLOY. Lockwood. Here are the details

https://www.assaabloyopeningsolutions.com.au/en/  

IDE details for Raw description after on-boarding the device. In IDE it shows Yale product.  For outsiders/certification its Lockwood.

Let us know for any changes/corrections.

Reply:
The second value in the rawDescription is the profile (0104). The third value (000A) is the device ID which you do not need to included in the fingerprint.

The deviceJoinName should be `Lockwood` according to that website.